contracts/actions/compoundV3/CompV3Claim.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+import '../ActionBase.sol';
+import './helpers/CompV3Helper.sol';
+import './helpers/MainnetCompV3Addresses.sol';
+import '../../interfaces/compoundV3/IComet.sol';
+import '../../interfaces/compoundV3/ICometRewards.sol';
+import '../../interfaces/IERC20.sol';
+
+/// @title Claims Comp reward for the specified user
+contract CompV3Claim is ActionBase, CompV3Helper {
+    struct Params {
+        address src;
```

Comment:
maybe change param name to onBehalf or something similar, src isn't descriptive enough